Readme.md

```diff
@@ -14,67 +14,67 @@ $ clib install clibs/buffer
 ## API
 
 ```c
-buffer_t *
-buffer_new();
+clib_buffer_t *
```

Comment:
why change the structure name and the following other functions? it would break to much things before.